CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features
 
 - Handle `monitor`/`check_in` in client reports and rate limiter ([#3096](https://github.com/getsentry/sentry-java/pull/3096))
+- Startup profiling 1 - Decouple Profiler from Transaction ([#3101](https://github.com/getsentry/sentry-java/pull/3101))
```

Comment:
So do you wanna ship this independently from the other PRs? If not, I'd just stack them and merge with only a single changelog entry